app/src/main/java/com/sunasterisk/fooddaily/utils/ImageUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.sunasterisk.fooddaily.utils
+
+import android.content.Context
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+
+fun applyGlide(
+    context: Context,
+    url: String?,
+    requestOptions: RequestOptions,
+    resource: Int,
+    imageView: ImageView
+) {
+    Glide.with(context)
+        .load(url)
+        .apply(requestOptions)
+        .placeholder(resource)
+        .into(imageView)
+}
```

Comment:
Sai ý tưởng rồi 
https://kotlinlang.org/docs/reference/extensions.html
Đặt tên file là ImageViewExt

Reply:
Ông nền tìm hiểu về Extensions trong Kotlin nhé, 
Đây cũng là 1 ví dụ: 
https://github.com/ricardoAntolin/androidKotlinCleanArchitecture/blob/master/app/src/main/kotlin/com/rantolin/cleanarchitecture/utils/Extensions.kt
```suggestion
fun ImageView.load(
    url: String,
    requestOptions: RequestOptions? = null ,
    placeholder: Int? = null
) {
    Glide.with(context)
        .load(url)
        .apply {
            requestOptions?.let{
                apply(it)
            }
            placeholder?.let{
                placeholder(it)
            }
        }
        .into(this)
}
```

Vì đoạn code 
``` Kotlin
a?.let {
    doSomething(it)
}
```
có thể viết gọn lại thành` a?.let(::doSomething)`

nên hàm bên trên có thể viết gọn lại hơn nữa